src/Shared/colors.ts

```diff
@@ -4,6 +4,7 @@ export const colors = {
   coastalMeadow: "#2ab78f",
   buoyYellow: "#ffcd22",
   whatOrange: "#ffae6d",
+  brighterOrange: "#fc8042",
```

Comment:
Is this from the NERACOOS styles?

Reply:
No its not from NERACOOS styles, but to me, it seems like none of the current colors have enough of a contrast to be effective as a third color in the graph:
This is what it looks like with neracoos' `whatOrange`:
![Screenshot 2024-12-05 at 12 59 24 PM](https://github.com/user-attachments/assets/c21b08b5-d637-4762-81d6-4446e58cdf45)
The best contrast for the colors we have is `neracoosBlue` but that just looks like a bolder version of the "Observed" timeseries. 
![Screenshot 2024-12-05 at 1 00 20 PM](https://github.com/user-attachments/assets/f6ee7ae3-24a0-474b-9285-606494845dc1)

The `brighterOrange` that I added looks like this:
![Screenshot 2024-12-05 at 1 01 24 PM](https://github.com/user-attachments/assets/06f73add-e6e7-4542-8ab2-fbc943679479)

Happy to change it to whatever you think is best! Just want it to be clear what timeseries is which (especially for folks who might be visually impaired)

Reply:
Is it cycling through the colors or are you picking them? 

What about using different dash styles for predicted vs forecast? That's what we are doing on the platform forecasts https://github.com/gulfofmaine/Neracoos-1-Buoy-App/blob/b90c86d5ea4056824cb3ea2889b6139aaf296f75/src/Features/ERDDAP/Platform/Forecasts/Page/index.tsx#L105

<img width="1391" alt="image" src="https://github.com/user-attachments/assets/2d6aa0b8-5073-41f2-aeb1-1cd2f2150360">


https://mariners.neracoos.org/platform/E01/forecast/air_temperature Though now I'm thinking we should probably make the observed solid and the forecasts dashed there

Hohonu uses dashes for their forecasts and dots for their observations

<img width="869" alt="image" src="https://github.com/user-attachments/assets/a6113819-bd26-4fc8-8188-51bbfe5b2089">

CO-OPs is solid for both

<img width="1066" alt="image" src="https://github.com/user-attachments/assets/a6f33830-0063-40b6-9a2f-5052a97be445">

We show prelim (but out of date because I gonna fix something with it...) data dashed on the Climate dashboard 
<img width="1382" alt="image" src="https://github.com/user-attachments/assets/0f976809-4407-4f76-9fce-ff7d474bbd82">


- https://www.highcharts.com/demo/highcharts/line-forecast
- https://api.highcharts.com/highcharts/series.line.dashStyle
- https://api.highcharts.com/class-reference/Highcharts#.DashStyleValue